src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -3,127 +3,101 @@
+                                                    'WindowsServer2016Datacenter',
+                                                    'WindowsServer2016DatacenterwithContainers',
+                                                    'UbuntuServer1604']),
+                   default='WindowsServer2016Datacenter', options_list=['--vm-os', '--os'],
+                   help='The Operating System of the VMs that make up the cluster.')
+        c.argument('node_type', help='the Node type name.')
+
+    with self.argument_context('sf cluster list') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type, id_part=None, help='The resource group name')
+
+    with self.argument_context('sf client certificate') as c:
+        c.argument('certificate_common_name', help='client certificate common name.')
+        c.argument('admin_client_thumbprints', nargs='+', help='Space separated list of client certificate thumbprint that only has admin permission, ')
+        c.argument('certificate_issuer_thumbprint', help='client certificate issuer thumbprint.')
+
+    with self.argument_context('sf cluster certificate') as c:
+        c.argument('thumbprint', help='The cluster certificate thumbprint to be removed')
+
+    with self.argument_context('sf cluster client-certificate') as c:
+        c.argument('is_admin', help='Client authentication type.')
+        c.argument('certificate_issuer_thumbprint', help='client certificate issuer thumbprint.')
+        c.argument('certificate_common_name', help='client certificate common name.')
+        c.argument('admin_client_thumbprints', nargs='+', help='client certificate thumbprint that only has admin permission.')
+        c.argument('readonly_client_thumbprints', nargs='+', help='Space separated list of client certificate thumbprint that has read only permission.')
+
+    with self.argument_context('sf cluster client-certificate add') as c:
+        c.argument('thumbprint', help='client certificate thumbprint.')
+
+    with self.argument_context('sf cluster client-certificate remove') as c:
+        c.argument('thumbprints', nargs='+', help='A single or Space separated list of client certificate thumbprint(s) to be remove.')
+
+    with self.argument_context('sf cluster node') as c:
+        c.argument('number_of_nodes_to_add', help='number of nodes to add.')
+        c.argument('number_of_nodes_to_remove', help='number of nodes to remove.')
+
+    with self.argument_context('sf cluster durability') as c:
+        c.argument('durability_level', arg_type=get_enum_type(['Bronze', 'Silver', 'Gold']), help='durability level.')
+
+    with self.argument_context('sf cluster setting') as c:
+        c.argument('parameter', help='parameter name')
+        c.argument('section', help='section name')
+        c.argument('value', help='Specify the value')
+        c.argument('settings_section_description', help='Specify the value')
```

Comment:
"Specify the value"?  Should be something better here.

Reply:
Leaving as is as that's what they have currently.